### PR TITLE
[luci-interpeter] Add helper for import without copying

### PR DESCRIPTION
**Pull request by m-bronnikov**
This commit adds helper skeleton for import without constant copying.

ONE-DCO-1.0-Signed-off-by: Maksim Bronnikov <max120199@gmail.com>

------------------

For: #8042
Draft: #8190
First part of: #8315

### CODE REVIEW

**Review by seanshpark — APPROVED** [review]
LGTM

File: compiler/luci-interpreter/src/CMakeLists.txt [hunk 1]
```diff
@@ -13,13 +13,16 @@ set(LUCI_INTERPRETER_BINARY "luci_interpreter${LUCI_INTERPRETER_SUFFIX}")
...
+set(LUCI_INTERPRETER_IMPORT "luci_interpreter_import${LUCI_INTERPRETER_SUFFIX}")
 
 add_subdirectory(core)
 message(STATUS "LUCI INTERPRETER CORE")
 add_subdirectory(kernels)
 message(STATUS "LUCI INTERPRETER KERNELS")
 add_subdirectory(loader)
 message(STATUS "LUCI INTERPRETER LOADER")
+add_subdirectory(import)
+message(STATUS "LUCI IMPORT IMPORT")
```
Comment (binarman) [inline compiler/luci-interpreter/src/CMakeLists.txt:25]: is it intentionally Import import?

Comment (m-bronnikov) [reply compiler/luci-interpreter/src/CMakeLists.txt:25]: Ahh, this is a typo =(

Comment (m-bronnikov) [reply compiler/luci-interpreter/src/CMakeLists.txt:25]:
```suggestion
message(STATUS "LUCI INTERPRETER IMPORT")
```

**Review by binarman — APPROVED** [review]
LGTM!

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**